browser/model/che.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import path from 'path';
+import InstallableItem from './installable-item';
+import Installer from './helpers/installer';
+import Platform from '../services/platform';
+
+class CheInstall extends InstallableItem {
+  constructor(installerDataSvc, url) {
+    super(CheInstall.KEY, url, '', '', installerDataSvc, false);
+    this.addOption('install', this.version, '', false);
+  }
+
+  static get KEY() {
+    return 'che';
+  }
+
+  installAfterRequirements(progress, success, failure) {
+    progress.setStatus('Installing');
+    let installer = new Installer(this.keyName, progress, success, failure);
+    return Promise.resolve().then(()=>{
+      return installer.exec('minishift config set memory 5GB && minishift addon enable che', this.createEnvironment());
```

Comment:
@LalatenduMohanty looks like we are enabling during installation. I think we need to just set memory when user click to install Che. and point to documentation to apply che.

Reply:
/cc @dgolovin

Reply:
@budhrg @LalatenduMohanty Why would we call it Eclipse Che Installer then?

Reply:
@dgolovin is right. Looks good to me. @budhrg the idea of installer is to make the process super easy for user. So this is the right thing to do as Che would be a separate entity in installer.